src/Fantomas/FormatConfig.fs

```diff
@@ -15,7 +15,18 @@ type FormatConfig =
       /// The column where we break to new lines
       PageWidth : Num
       SemicolonAtEndOfLine : bool
-      SpaceBeforeArgument : bool
+      SpaceBeforeUnitArgumentInUppercaseFunctionCall : bool
+      SpaceBeforeUnitArgumentInLowercaseFunctionCall : bool
+      SpaceBeforeParenthesesArgumentInUppercaseFunctionCall : bool
+      SpaceBeforeParenthesesArgumentInLowercaseFunctionCall : bool
+      SpaceBeforeUnitParameterInUppercaseFunctionDefinition : bool
+      SpaceBeforeUnitParameterInLowercaseFunctionDefinition : bool
+      SpaceBeforeParenthesesInUppercaseFunctionDefinition : bool
+      SpaceBeforeParenthesesInLowercaseFunctionDefinition : bool
+      SpaceBeforeUnitParameterInUppercaseClassConstructor : bool
+      SpaceBeforeUnitParameterInLowercaseClassConstructor : bool
+      SpaceBeforeParenthesesParameterInUppercaseClassConstructor : bool
+      SpaceBeforeParenthesesParameterInLowercaseClassConstructor : bool
```

Comment:
why use the word "parameter" in some places and the word "argument" in others?

Reply:
https://stackoverflow.com/questions/156767/whats-the-difference-between-an-argument-and-a-parameter

Reply:
interesting 🤔